### PR TITLE
Added Chess Game

### LINKED ISSUE #198: CHESS GAME
Hey I am a GSSOC'22 contributor.
I would like to add a chess game project using HTML, CSS, JavaScript. Kindly assign me this issue.

### PR DESCRIPTION
**Pull request by rittikadeb**
### 👨‍💻 Changes proposed
Added a new Chess Game with a flipboard feature.

### ✅ Check List (Check all the applicable boxes)

- [x] My code doesn't break any part of the project (Zero Octave-Javascript-Projects).
- [x] This PR does not contain plagiarized content.
- [x] My Addition/Changes works properly and matches the overall repo pattern.
- [x] The title of my pull request is a short description of the requested changes.

### 📷 Screenshots
image

### CODE REVIEW

**Comment by NOiR-07** [comment]
You can add your project in the index file of website just like other projects

**Comment by NOiR-07** [comment]
your css isn't working properly, please check

**Comment by rittikadeb** [comment]
https://user-images.githubusercontent.com/76259897/156886089-a81f3b35-595e-4ed0-b720-2602e3ff391d.mp4

The code is working. I have checked it and this is the working video.

**Comment by NOiR-07** [comment]
@sreelakshmig009 @Astrodevil  can you check on your local machine too

File: Public/chess.html [hunk 1]
```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+	<link rel="stylesheet" type="text/css" href="chess_game.css">
```
Comment (Astrodevil) [inline Public/chess.html:5]: This linkage is wrong.  It should be <link rel="stylesheet" href="../assets/css/chess_game.css" />

Comment (sreelakshmig009) [reply Public/chess.html:5]: The link to the stylesheet needs to be changed because your chess_game.css file is stored in the assets folder

Comment (rittikadeb) [reply Public/chess.html:5]: Sure I will change it. Sorry for the inconveniences.

**Comment by Astrodevil** [comment]
@rittikadeb Is everything running fine? I suggested you check all linkages but you only updated the CSS linkage. You are running the project in your own folder but files should be linked according to the directory of this repo before merging.

**Comment by rittikadeb** [comment]
Hey I have made the changes and also added the final project in the index.html. Please review it.

**Comment by rittikadeb** [comment]
I cannot find the conflict.

**Comment by NOiR-07** [comment]
You have copy-pasted the code in the index file, don't do that, just you need to make a card class and add your project like others did

**Pull request closed without merging**